src/adapter/threads.ts

```diff
@@ -1157,8 +1159,24 @@ export class Thread implements IVariableStoreDelegate {
     slot(output);
   }
 
-  _onThreadPaused(details: IPausedDetails) {
+  async _onThreadPaused(details: IPausedDetails) {
```

Comment:
Doesn't look like `_onThreadPaused` is being awaited anywhere now that it's async. Is this intended?